includes/admin/class-wc-stripe-inbox-notes.php

```diff
@@ -0,0 +1,69 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+use Automattic\WooCommerce\Admin\Notes\WC_Admin_Note;
+use Automattic\WooCommerce\Admin\Notes\WC_Admin_Notes;
+
+/**
+ * Class that adds inbox notifications.
+ *
+ * @since 4.5.4
+ */
+class WC_Stripe_Inbox_Notes {
+	const NOTE_NAME = 'stripe-apple-pay-domain-verification';
+
+	// TODO Call this only after domain verification has a chance to run
+	public static function notify_of_apple_pay_domain_verification_if_needed() {
+		if ( ! class_exists( 'Automattic\WooCommerce\Admin\Notes\WC_Admin_Notes' ) ) {
+			return;
+		}
+
+		if ( ! class_exists( 'WC_Data_Store' ) ) {
+			return;
+		}
+
+		$stripe_settings       = get_option( 'woocommerce_stripe_settings', array() );
+		$domain_flag_key       = 'apple_pay_domain_set';
+		$verification_complete = isset( $stripe_settings[ $domain_flag_key ] ) && 'yes' === $stripe_settings[ $domain_flag_key ];
+
+		// Only show if in the US.
```

Comment:
Why?

Reply:
Good question – it was in the note I was adapting, and I thought it was desired but I may have just been conflating a different requirement (now tracked in #1351) with this more technical case.

Reply:
Probably should be removed, right?

Reply:
Done in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1337/commits/3bbdf691c8fef5723bf7641f18ddea62a98a0ed3 – thanks